### ISSUE TITLE
Comments for Rule efbfc7: Text content that changes automatically can be paused, stopped or hidden

### ISSUE THREAD

**Issue by kengdoj** [act-rules/act-rules.github.io#1627]
[Rule efbfc7](https://act-rules.github.io/rules/efbfc7)

### Accessibility Requirements Mapping
- Non-conformance is listed as not required for WCAG conformance. [WCAG 2.1 5.2 Conformance Requirements](https://www.w3.org/TR/WCAG21/#conformance-reqs) states: In order for a Web page to conform to WCAG 2.1, all of the following conformance requirements must be satisfied: 5.2.5 Non-interference.

### Assumptions
- 3rd bullet from bottom uses "mechanism". Should be instrument?
- Should it be added that the instrument meets all SC? 
  - The [definition of mechanism](https://www.w3.org/TR/WCAG20/) has a "Note 2: The mechanism needs to meet all success criteria for the conformance level claimed." The conformance of the mechanism affects the outcome of 2.2.2.

**Comment by Jym77** [comment]
> * Non-conformance is listed as not required for WCAG conformance. [WCAG 2.1 5.2 Conformance Requirements](https://www.w3.org/TR/WCAG21/#conformance-reqs) states: In order for a Web page to conform to WCAG 2.1, all of the following conformance requirements must be satisfied: 5.2.5 Non-interference.

This is actually a bug in the website code generation (https://github.com/act-rules/act-rules-web/issues/235) The rule file itself has the correct `forConformance: true`. I kinda hope this should solve automatically when moving to new website (or hosted with the rest of WAI stuff).

> * 3rd bullet from bottom uses "mechanism". Should be instrument?

Yes, most definitely.

> * Should it be added that the instrument meets all SC?

We've discussed this and ended up deciding that no but adding notes. See #1524 and https://www.w3.org/2021/04/08-act-r-minutes.html#item03 for the discussion; and #1632 for the proposed (global) fix that should probably be copied also in that rule.

**Comment by carlosapaduarte** [comment]
#1773 replaces mechanism with instrument
The other two points have been addressed elsewhere, since the current rule no longer suffers from those
